### PR TITLE
Initial Page Override Operations 

### PR DESCRIPTION
**Pull request by nheyland-foreflight**

### CODE REVIEW

File: server/modules/storage/git/storage.js [hunk 1]
```diff
@@ -298,6 +298,22 @@ module.exports = {
   async created(page) {
     WIKI.logger.info(`(STORAGE/GIT) Committing new file [${page.localeCode}] ${page.path}...`)
     let fileName = `${page.path}.${pageHelper.getFileExtension(page.contentType)}`
+
+    // Generate branch name for this new page
+    const branchName = `feature/new-${page.path.replace(/\//g, '-')}`
```
Comment (zbrands-foreflight) [inline server/modules/storage/git/storage.js:303]: This is a tiny nit, but can the name for the branch name include `-page` at the end? 
example: `feature/new-home-page`

File: server/modules/storage/git/storage.js [hunk 2]
```diff
@@ -320,6 +342,21 @@ module.exports = {
   async updated(page) {
     WIKI.logger.info(`(STORAGE/GIT) Committing updated file [${page.localeCode}] ${page.path}...`)
     let fileName = `${page.path}.${pageHelper.getFileExtension(page.contentType)}`
+
+    const branchName = `feature/new-${page.path.replace(/\//g, '-')}`
```
Comment (zbrands-foreflight) [inline server/modules/storage/git/storage.js:346]: Same as Line 303 (add `-page`)

**Review by zbrands-foreflight — COMMENTED** [review]
Small request, but not a blocker 👍

File: package.json [hunk 3]
```diff
@@ -270,7 +270,7 @@
     "jest": "26.6.1",
     "js-beautify": "1.13.5",
     "js-cookie": "2.2.1",
-    "mermaid": "8.8.2",
+    "mermaid": "11.2.1",
```
Comment (zbrands-foreflight) [inline package.json]: Does this need to be reverted back to `8.8.2`?

**Comment by zbrands-foreflight** [comment]
Does this PR need an approval/merged?

**Comment by nheyland-foreflight** [comment]
> Does this PR need an approval/merged?

Unfortunately I hit a really big issue with the git stuff and need to re-evaluate it. Ultimately this will need to be merged but for now, more work required.

**Pull request merged**